### PR TITLE
Don't render with Gemini on initial render of SchemaForm

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/components/SchemaForm.js [hunk 1]
```diff
@@ -252,10 +257,18 @@ class SchemaForm extends React.Component {
       );
     });
 
+    if (!this.state.renderGemini) {
```
Comment (mlunoe) [inline src/js/components/SchemaForm.js]: Could we get a comment about why this is necessary? Something like: 'Gemini measures the height/width which causes the animation to cancel'

**Comment by mlunoe** [comment]
After [this](https://github.com/dcos/dcos-ui/pull/138#discussion_r62591805) comment I think it looks good: ⛵

File: src/js/components/SchemaForm.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/SchemaForm.js]: Done!

**Pull request merged**